pkgs/applications/science/biology/flywheel-cli/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, stdenv
+, fetchurl
+, unzip
+}:
+
+let
+  inherit (stdenv.targetPlatform) system;
+  throwSystem = throw "Unsupported system: ${system}";
```

Comment:
I saw a few packages using this pattern, like [Slack](https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/networking/instant-messengers/slack/default.nix).